src/main/java/com/spotify/docker/client/messages/mount/Driver.java

```diff
@@ -24,19 +24,24 @@
 import static com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility.NONE;
 
 import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.google.auto.value.AutoValue;
 
 import com.google.common.collect.ImmutableMap;
 import java.util.Map;
 
+import javax.annotation.Nullable;
+
 @AutoValue
 @JsonAutoDetect(fieldVisibility = ANY, getterVisibility = NONE, setterVisibility = NONE)
 public abstract class Driver {
 
+  @Nullable
```

Comment:
I'm not really sure about the @Nullable on name.  Might there be an occasion when options are entered against the default driver? (I'm no Docker expert)